pkg/gui/services/custom_commands/session_state_loader.go

```diff
@@ -182,7 +182,12 @@ type SessionState struct {
 }
 
 func (self *SessionStateLoader) call() *SessionState {
-	var commit = commitShimFromModelCommit(self.c.Contexts().LocalCommits.GetSelected())
+	commit := commitShimFromModelCommit(self.c.Contexts().LocalCommits.GetSelected())
+	path := self.c.Contexts().Files.GetSelectedPath()
+
+	if path == "" {
+		path = self.c.Contexts().CommitFiles.GetSelectedPath()
+	}
```

Comment:
Same here; testing if path is empty doesn't make sense. We need to set SelectedPath to either Files.GetSelectedPath() or CommitFiles.GetSelectedPath(), depending on which view is focused.